layout/theme.liquid

```diff
@@ -69,6 +73,7 @@
         --gradient-base-accent-2: {% if settings.gradient_accent_2 != blank %}{{ settings.gradient_accent_2 }}{% else %}{{ settings.colors_accent_2 }}{% endif %};
 
         --page-width: {{ settings.page_width | divided_by: 10 }}rem;
+        --page-width-margin: {{ page_width_margin }}rem;
```

Comment:
Is the `assign` necessary or could the value be set inline?

`--page-width-margin: {% if settings.page_width == '1600' %}2{% else %}0{% endif %}rem;`